pkg/infra/metrics/metrics.go

```diff
@@ -343,6 +355,31 @@ func init() {
 		Namespace:  ExporterName,
 	})
 
+	MRenderingRequestCompleted = newCounterStartingAtZero(prometheus.CounterOpts{
+		Name:      "rendering_request_completed_total",
+		Help:      "counter for successfully rendered panels",
+		Namespace: ExporterName,
+	})
+
+	MRenderingRequestFailed = newCounterStartingAtZero(prometheus.CounterOpts{
+		Name:      "rendering_request_failed_total",
+		Help:      "counter for panels rendered with error",
+		Namespace: ExporterName,
+	})
```

Comment:
Think this should be one metric `rendering_request_total`, following our existing convention of `http_request_total` and having labels for success/failure.

Maybe not need `newCounterStartingAtZero`, but not sure why some have it and some others don't. Maybe @bergquist can bring some clarity to this?

Reply:
For http requests we use HTTP status codes (200/404/500). What kind of labels should we use here? success/error?